rpc/tests/rpc.rs

```diff
@@ -1,77 +1,90 @@
 #[cfg(test)]
 mod test {
-    use kakarot_rpc::eth_rpc::KakarotEthRpc;
-    use kakarot_rpc_core::lightclient::{MockStarknetClient, StarknetClientImpl};
-    use reth_primitives::{Address, Bytes};
-
-    use starknet::providers::jsonrpc::models::BlockId as StarknetBlockId;
+    use kakarot_rpc_utils::wiremock_utils::setup_rpc_server;
 
     #[tokio::test]
-    async fn when_call_block_number_return_ok() {
-        // Given
-        // Mock config, ethereum light client and starknet light client.
-        let mut starknet_lightclient_mock = config();
+    async fn test_eth_chain_id_is_ok() {
+        let (_, server_handle) = setup_rpc_server().await;
+        let client = reqwest::Client::new();
+        let res = client
+            .post("http://127.0.0.1:3030")
+            .body("{\"jsonrpc\": \"2.0\", \"id\": 1, \"method\": \"eth_chainId\", \"params\": [] }")
+            .header("content-type", "application/json")
+            .send()
+            .await
+            .unwrap();
 
-        // Set expect to testing RPC method
-        starknet_lightclient_mock
-            .expect_block_number()
-            .returning(|| Ok(1));
+        let _has_stop = server_handle.stop().unwrap();
```

Comment:
what are you testing here? no assertion is being made
are you checking that the .unwrap() does not panic?